apps/src/applab/designMode.js

```diff
@@ -549,6 +548,30 @@ designMode.onDuplicate = function (element, event) {
   return duplicateElement;
 };
 
+function duplicateScreen(element) {
+  let sourceScreen = $(element);
+  let newScreen = designMode.createScreen();
+  designMode.changeScreen(newScreen);
+
+  // Unwrap the draggable wrappers around the elements in the source screen:
+  let madeUndraggable = makeUndraggable(sourceScreen.children());
+
+  // Clone each child of the source screen into the new screen (with new ids):
+  sourceScreen.children().each(function () {
+    var clonedChild = $(this).clone(true)[0];
+    let elementType = elementLibrary.getElementType(clonedChild);
```

Comment:
Nit: I believe we can be using `const` throughout this new code instead of var/let

Reply:
👍